server/api/users.js

```diff
@@ -222,6 +222,14 @@ router.post('/users/new', apiLimiter, multer.single('avatarFile'), async (req, r
     if (email) {
       createObj.email = email;
       createObj.isEmailVerified = isEmailVerified;
+
+      if (isEmailVerified) {
+        await dbRef
+          .doc(user)
+          .collection('mission')
+          .doc('verifyEmail')
+          .set({ done: true }, { merge: true });
```

Comment:
please state with comment this is a hack, since we should rely on firestore trigger to mark the mission done, which in reality is too slow for new user UX...